packages/amplify-e2e-core/src/utils/test-regions.ts

```diff
@@ -14,3 +15,12 @@ export const isOptInRegion = (region: string): boolean => {
 
   return specificRegion.optIn;
 };
+
+export const isDataAPISupported = (region: string): boolean => {
+  const repoRoot = path.join(__dirname, '..', '..', '..', '..');
+  const supportedRegionsPath = path.join(repoRoot, 'scripts', 'e2e-test-regions.json');
+  const supportedRegions: TestRegion[] = JSON.parse(fs.readFileSync(supportedRegionsPath, 'utf-8'));
+  const specificRegion = supportedRegions.find((testRegion) => testRegion.name == region);
```

Comment:
nit: The 4 lines above can be extracted into a new function `loadTestRegions` as it has the same region file path and is also used by the `isOptInRegion`